examples/image-streamer/api300/golden_image.rb

```diff
@@ -0,0 +1,88 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative '../../_client_i3s' # Gives access to @client
+
+# Example: Create a golden image for an API300 Image Streamer
+# NOTE: This will create a golden images named like 'Golden_Image_1' and 'Golden_Image_2', then delete it.
```

Comment:
**This will create** ~~a~~ **golden images named** ~~like~~ **'Golden_Image_1' and 'Golden_Image_2'** ~~then delete it~~ **and then, it will delete them**.